src/Draft/Attachments/AttachmentField.vue

```diff
@@ -78,12 +78,12 @@ const emit = defineEmits(['delete', 'attached']);
 
 const field = computed(() => props.field as FieldSpec);
 const language = useLanguageStore();
-const progress = ref('width:0%');
+const progress = ref('width:0.9%');
 const uploading = ref(false);
 
 // This hooks to the requestObj onFileProgress event so that we can indicate upload progress.
 const onFileProgress = (event: any) => {
-  progress.value = 'width:' + Math.round((event.loaded * 100.0) / event.total) + '%';
+  progress.value = 'width:' + Math.round(event * 100) + '%';
```

Comment:
The progress was broken in the other attachment types since the function expects one argument a number on `onProgress`.  Look at the uploads via Axios https://github.com/story-cms/ui/blob/develop/src/Draft/Attachments/cloudinary-service.ts.  And what `onFileProgress` "receives" is a number.